src/iqm_client/iqm_client.py

```diff
@@ -171,10 +171,7 @@ class RunRequest(BaseModel):
     'quantum circuit to execute'
     settings: dict[str, Any] = Field(..., description='EXA settings node containing the calibration data')
     'EXA settings node containing the calibration data'
-    qubit_mapping: list[SingleQubitMapping] = Field(
-        ...,
-        description='mapping of logical qubit names to physical qubit names'
-    )
+    qubit_mapping: Optional[list[SingleQubitMapping]]
```

Comment:
```suggestion
    qubit_mapping: Optional[list[SingleQubitMapping]] = Field(
        None,
        description='mapping of logical qubit names to physical qubit names'
    )
```
We need the description field for the JSON schema.